tests/integration/cairo_path/contracts/starknet_project/basic_starknet_contract.cairo

```diff
@@ -0,0 +1,8 @@
+#[contract]
```

Comment:
Why does `mycontract.cairo` specifies `use external_lib...` but this file doesn't?

Reply:
It worked that way. Please, note that `mycontract` and `basic_starknet_contract` use different compilers so I guess the difference is there.